ts/lib/commands/precompile.ts

```diff
@@ -25,18 +25,25 @@ export default command({
       return;
     }
 
-    // prettier-ignore
-    await execa('tsc', [
-      '--allowJs', 'false',
-      '--noEmit', 'false',
-      '--rootDir', rootDir || this.project.root,
-      '--isolatedModules', 'false',
-      '--declaration',
-      '--declarationDir', outDir,
-      '--emitDeclarationOnly',
-    ], {
-      preferLocal: true
-    });
+    try {
+      // prettier-ignore
+      await execa('tsc', [
+        '--allowJs', 'false',
+        '--noEmit', 'false',
+        '--rootDir', rootDir || this.project.root,
+        '--isolatedModules', 'false',
+        '--declaration',
+        '--declarationDir', outDir,
+        '--emitDeclarationOnly',
+        '--pretty', 'true',
+      ], {
+        preferLocal: true,
+        all: true,
```

Comment:
Would you mind adding a comment here explaining why we want this? Will make maintenance easier. (I just went and read the `execa` docs to figure it out, figure it'll save us time in the future from doing the same.)

Reply:
Done! `execa` is one of those that I've used enough places that I've lost any good sense of which options are self-describing and which aren't 🙃

Reply:
I sympathize!